devtools/conda-envs/test_env.yaml

```diff
@@ -0,0 +1,27 @@
+name: test
+channels:
+  - conda-forge
+  - omnia
+  - ambermd
+dependencies:
+    # Base depends
+  - python
+  - pip
+
+    # Testing
```

Comment:
Add openforcefield

Reply:
Add openeye-toolkits and openeye, configure OE license. List instructions for how to update when it expires.